qa/L0_model_update/instance_update_test.py

```diff
@@ -278,6 +278,24 @@ def test_non_instance_config_update(self):
                                      batching=True)
         self.__unload_model(batching=True)
 
+    # Test passing new instance config via load API
+    def test_load_api_with_config(self):
+        # Load model with 1 instance
+        self.__load_model(1)
+        # Add 1 instance via the load API
+        new_config = "{\"name\": \"" + self.__model_name + "\",\n"
+        new_config += "\"backend\": \"python\",\n"
+        new_config += "\"max_batch_size\": 0,\n"
+        new_config += "\"input\": {\"name\": \"INPUT0\", \"data_type\": \"TYPE_FP32\", \"dims\": -1},\n"
+        new_config += "\"output\": {\"name\": \"OUTPUT0\", \"data_type\": \"TYPE_FP32\", \"dims\": -1},\n"
+        new_config += "\"instance_group\": {\"count\": 2, \"kind\": \"KIND_CPU\"}}"
```

Comment:
Can you use `dict` and `json.dumps(dict)` instead?

Reply:
Yes. Updated: [Use dictionary for json config](https://github.com/triton-inference-server/server/pull/5937/commits/6d9c4f4fbcb9354240d6b38c0f53079dbc5b3799)